internal/loadbalancebenchmark/config.go

```diff
@@ -0,0 +1,120 @@
+package loadbalancingbenchmark
+
+import (
+	"fmt"
+	"time"
+)
+
+type ClientGroup struct {
+	Cnt    int
+	Rps    int
+	LType  ListType
+	LUType ListUpdaterType
+}
+
+type LatencyConfig struct {
+	P50  time.Duration
+	P90  time.Duration
+	P99  time.Duration
+	P100 time.Duration
+}
+
+type ServerGroup struct {
+	Cnt           int
+	Type          MachineType
```

Comment:
as discussed in person, let's make this a generic label/string to give people more flexibility

Reply:
now this field could `Name` and could be set to any string values.